addons/mail/static/src/components/chat_window_manager/tests/chat_window_manager_tests.js

```diff
@@ -2,40 +2,22 @@
 
 import { makeDeferred } from '@mail/utils/deferred/deferred';
 import {
-    afterEach,
     afterNextRender,
     beforeEach,
     nextAnimationFrame,
-    start,
 } from '@mail/utils/test_utils';
 
+import { browser } from "@web/core/browser/browser";
 import { file, dom } from 'web.test_utils';
+import { patchWithCleanup } from "@web/../tests/helpers/utils";
+
 const { createFile, inputFiles } = file;
 const { triggerEvent } = dom;
 
 QUnit.module('mail', {}, function () {
 QUnit.module('components', {}, function () {
 QUnit.module('chat_window_manager', {}, function () {
-QUnit.module('chat_window_manager_tests.js', {
-    beforeEach() {
-        beforeEach(this);
-
-        this.start = async params => {
-            const { afterEvent, env, widget } = await start(Object.assign(
-                { hasChatWindow: true, hasMessagingMenu: true },
```

Comment:
From my understanding, now all tests have chat window manager and messaging menu deployed?

It was nice to know which tests actually needed them.
Discuss does the same, but at least there's the `openDiscuss()` function to let us know it's needed.

Could be necessary to always deploy them, due to how it still relies on legacy code and how the `createWebClient()` function works.

The intent is to put most logic in the discuss components that will be turned into models. Each test should explicitly change the state in models to affect the UI of discuss code, such as to display the messaging menu component or, chat window components, and or discuss component.

It's alright to keep current approach for the time being, as long as we know what the goals are and what we need to plan in the future to get in that direction.